NSS-Day-Cohort-53/tabloid-full-stack-always-puffy-impalas
# User is now able to delete tag

#### Changes Made
1. Added file `TagDelete.js` to `components` directory.
2. Modified file `tagManager.js` to include `deleteTag` and functionality.
3.  Modified `ApplicationViews.js` to include tag/delete route
4. Modified `TagList.js` to include a delete button 
5. Modified `TagRepository` to include deleteTag 
6. Modified `ITagRepository` to include deleteTag
7. Modified TagController to include delete functionality
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout HP-deleteTag
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When logged in, click the link to the tag view
    > On the tag view, each tag should have an edit and delete button
    > Click the delete button and the user should be redirected to a view which confirms if they would like to delete the tag
   > Upon confirmation to delete, user should be redirected to tags list
   > If the user chooses to not delete the tag, they should be redirected back to the tag list
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.

Closes #33
